Jenkinsfile

```diff
@@ -134,24 +125,17 @@ pipeline {
 
   post {
     // This is the order that the methods are run. {always->success/abort/failure/unstable->cleanup}
-    always {
-      echo "always"
-      script { if ( !CI_STATE.SELF.RUN_BUILD || !CI_STATE.SELF.RUN_TESTS ) { currentBuild.result = "UNSTABLE"}}
-    }
-    success {
-      echo "success"
-      script { lib_Status.set("SUCCESS", 'MCUBOOT', CI_STATE) }
-    }
-    aborted {
-      echo "aborted"
-      script { lib_Status.set("ABORTED", 'MCUBOOT', CI_STATE) }
-    }
-    unstable {
-      echo "unstable"
-    }
+    always { script {
+      lib_Status.set( "${currentBuild.currentResult}", 'MCUBOOT', CI_STATE)
+      if ( !CI_STATE.SELF.RUN_BUILD || !CI_STATE.SELF.RUN_TESTS ) { currentBuild.result = "UNSTABLE"}
+    }}
+    // Add if needed
+    // success {}
+    // aborted {}
+    // unstable {}
+    // failure {}
```

Comment:
This one already exists, so does that mean this could be un-commented to get two blocks run in that order?

Reply:
no, I removed the line.